rewrite-java/src/main/resources/META-INF/rewrite/eureka.yml

```diff
@@ -24,7 +24,7 @@ styleConfigs:
       layout:
         - import all other imports
         - <blank line>
-        - import javax.*
-        - import java.*
+        - import javax.* withSubpackages true
```

Comment:
the naming is just a placeholder to represent the IntelliJ import option:

- [ ] With Subpackages

Questions:
Are there any preferences about the wording?

- `with sub packages` | `without sub packages` | `withSubpackages true`
- `withSubpackages` | `with sub packages` | `with sub-packages`

Note: sub-packages will be necessary for issue 1191.
sub-packages are true by default in IntelliJ's import layout style, but false by default when explicit package folding is provided.